plugins/mcreator-localization/help/ru_RU/villagerprofession/profession_block.md

```diff
@@ -0,0 +1 @@
+Выберите блок, позволяющий крестьянину получить данную профессию.
```

Comment:
Translation of the suffix of the original help tip:
```
Если другой мод регистрирует выбранный блок для своей профессии либо в других целях,
то профессия, определяемая здесь, не появится в игре. Поэтому рекомендуется тщательно проверять,
что указанный блок используется только этой профессией.
```

Reply:
Fixed!